app/javascript/controllers/textarea_autogrow_controller.js

```diff
@@ -2,41 +2,28 @@ import { Controller } from '@hotwired/stimulus'
 
 export default class extends Controller {
 
-  initialize () {
+  initialize() {
     this.onResize = undefined
-    this.resizeDebounceDelayValue = 100
     this.autogrow = this.autogrow.bind(this)
   }
 
-  connect () {
+  connect() {
     this.element.style.overflow = 'hidden'
-    this.onResize = this.resizeDebounceDelayValue > 0 ? debounce(this.autogrow, this.resizeDebounceDelayValue) : this.autogrow
-
     this.autogrow()
 
     this.element.addEventListener('input', () => this.autogrow())
-    window.addEventListener('resize', () => this.onResize())
+    window.addEventListener('resize', () => this.debouncedAutogrow())
   }
 
-  disconnect () {
+  disconnect() {
     this.element.removeEventListener('input', () => this.autogrow())
-    window.removeEventListener('resize', () => this.onResize())
+    window.removeEventListener('resize', () => this.debouncedAutogrow())
```

Comment:
This disconnect won't work because the anonymous function is different from the one on line 15. From the way you have it implemented, I believe you can just do this:

```suggestion
    window.removeEventListener('resize', this.debouncedAutogrow)
```

(and match above)